projects/plugins/jetpack/modules/stats/class-jetpack-stats-upgrade-nudges.php

```diff
@@ -189,27 +305,22 @@ public static function render( $callback ) {
 		self::print_styles();
 		self::print_header();
 		if ( ! self::has_security_plan() ) {
-			self::print_item( __( 'Security', 'jetpack' ), 'products/product-jetpack-backup.svg' );
+			self::print_security();
 			if ( ! self::is_backup_active() ) {
-				self::print_item( __( 'Backup', 'jetpack' ), 'products/product-jetpack-backup.svg', true );
+				self::print_backup();
 			}
 			if ( ! self::is_scan_active() ) {
-				self::print_item( __( 'Scan', 'jetpack' ), 'products/product-jetpack-scan.svg', true );
+				self::print_scan();
 			}
 			if ( self::is_akismet_active() ) {
```

Comment:
We want to check whether Akismet is not active:

```suggestion
			if ( ! self::is_akismet_active() ) {
```